src/slider.scss

```diff
@@ -16,9 +16,13 @@ $handle-hit-area-mobile-dimensions: 2.75rem;
 
   min-width: 4rem;
   max-width: 100%;
-  padding: 1rem 0;
+  padding: 1rem 0.625rem;
   position: relative;
 
+  &--lg-x-paddings {
```

Comment:
Please rename it to just `&--lg`. You can add in the documentation (the text above the example for mobile mode something like:
"To add horizontal paddings of 0.625rem use the <code>fd-slider--lg</code> modifier class".